starter-code/src/functions-and-arrays.js

```diff
@@ -1,23 +1,73 @@
+
 // Find the maximum
 
+function maxOfTwoNumbers (var1, var2) {
+  if (var1 > var2){
+    return var1;
+  } else {
+    return var2;
+  }
+}
+
+var max = maxOfTwoNumbers(10, 6);
+console.log(max);
+
 // Finding Longest Word
+
+function findLongestWord (words) {
+  var longestWord="";
+  if (words.length === 0){
+    return;
+  }
+
+  for(var i=0; i < words.length; i++){
+    if (longestWord.length < words[i].length){
+        longestWord = words[i];
+    }else if(longestWord.length === words[i].length){
+        var morelongestWords = [];
+        morelongestWords.push(words[i] , longestWord);
```

Comment:
Good job thinking about possibilities, but this approach has two problems:
 - It only works if there are two longests words. If you find a third one, you will lose the previous results and store only the first and third longest words.
 - You only return the first longest word (line 31), so this piece of code (26-29) never changes the output.